.pylintrc

```diff
@@ -81,6 +81,7 @@ disable=missing-docstring,
       invalid-overridden-method, # temp-pylint-upgrade 
       missing-module-docstring, # temp-pylint-upgrade
       import-error, # needed as a workaround as reported here: https://github.com/open-telemetry/opentelemetry-python-contrib/issues/290
+      consider-using-f-string, # needed flexibility for string formatting
```

Comment:
Don't add this at the global level. Just disable pylint for the line where you need flexibility. This is useful rule